Product Designer/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+       <title>
+            Product Designer
+       </title>
+       <link rel="stylesheet" href="/Product Designer/assets/css/styles.css" />
+    </head>
+        <body>
+        <!-- START : HEADER SECTION -->
+        <header class="mailSection">
+            <h6>shruthi.bhatt.siriuscom.com</h6>
+        </header>
+        <!-- END :  HEADER SECTION -->
+
+        <!-- START : NAME SECTION -->
+        <section class="nameSection">
+            <h1 class="name">
+                <span>Shruthi</span> Bhatt
+            </h1>
+            <h2 class="role">
+                a <span>product</span> designer
+                <h6 class="mobileEmail">shruthi.bhatt.siriuscom.com</h6>
+            </h2>
+        </section>
+        <!-- END : NAME SECTION -->
+
+        <!-- START : PRODUCTS SECTION -->
+        <div class="productsContainer">
+            <img src="/Product Designer/assets/Images/rawpixel-769308-unsplash.png" alt="Product Image" />
+            <img src="/Product Designer/assets/Images/fancycrave-329196-unsplash.png" alt="Product Image" />
+            <img src="/Product Designer/assets/Images/nadine-shaabana-1112616-unsplash.png" alt="Product Image" />
+            <img src="/Product Designer/assets/Images/fikri-rasyid-689579-unsplash.png" alt="Product Image" />
+            <img src="/Product Designer/assets/Images/marc-mueller-654355-unsplash.png" alt="Product Image" />
+            <img src="/Product Designer/assets/Images/jeff-sheldon-3219-unsplash.png" alt="Product Image" />
+        </div>
+        <!-- END : PRODUCT SECTION -->
+
+        <!-- START : LOGO SECTION -->
+        <section class="logoContainer">
+            <div class="logoTitle">Big names to <span>show off</span> </div>
+            <div class="logoRow">
+                <img src="/Product Designer/assets/Images/spotify.png" alt="Spotify image" /> 
+                <img src="/Product Designer/assets/Images/audi.png" alt="Audi image" />
+                <img id ="adidas" src="/Product Designer/assets/Images/adidas.png" alt="adidas image" />
+                <img src="/Product Designer/assets/Images/lufthanza.png" alt="lufthanza image" />
+                <img src="/Product Designer/assets/Images/ikea.png" alt="ikea image" />
+                <img src="/Product Designer/assets/Images/montblanc.png" alt="montblanc" />
+            </div>
+        </section>
+        <!-- END : LOGO SECTION -->
+
+        <!-- START : FOOTER SECTION -->
+        <footer class="footer">
+            <div class="breakLines">
+                <span class="shortLine"></span>
+                <span class="longLine"></span>
+            </div>
+            <div class="iconsSection">
+                <img src="/Product Designer/assets/Images/facebook.png" alt="" />
```

Comment:
You should add alt values for images for fallBack logic

Reply:
Added alt values